src/resty/aws/init.lua

```diff
@@ -78,6 +78,66 @@ local aws_config = {
   api = {},
 }
 
+local detect_toxic_content_spec = {
```

Comment:
lua table representation of ToxicContent operation.

It can be found here too:
https://github.com/aws/aws-sdk-js-v3/blob/0899e938946741970e9bb4ad27c4ec8966ccbc09/codegen/sdk-codegen/aws-models/comprehend.json#L4977